src/pages/Facility/settings/chargeItemDefinitions/ChargeItemDefinitionDetail.tsx

```diff
@@ -136,12 +137,17 @@ export function ChargeItemDefinitionDetail({
   }
 
   return (
-    <Page title={chargeItemDefinition.title}>
+    <Page title={chargeItemDefinition.title} hideTitleOnPage={true}>
```

Comment:
```suggestion
    <Page title={chargeItemDefinition.title} hideTitleOnPage>
```

 a boolean prop without a value defaults to true